storagenode/trust/file_source_test.go

```diff
@@ -0,0 +1,162 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package trust
```

Comment:
Prefer `package trust_test` when testing, it makes some design mistakes more obvious.

Reply:
Done.